config/README.md

```diff
@@ -25,6 +25,10 @@ The config files contain a features object that can be used to determine whether
 
 Please make sure to add new feature flags alphabetically so they are easy to find and add any new feature flags in all config files (client.json, development.json, production.json, horizon.json, stage.json, test.json, wpcalypso.json).
 
+You can search for feature flags by partial string or regular expression with the [bin/feature-search.js](bin/feature-search.js) too.
+
+Run `node bin/feature-search.js` from the root calypso directory to see example searches.
```

Comment:
Might it be worth adding this to the `scripts` section in `package.json` so it can be run using `yarn feature-search plugin`? We could document both sets of syntax, but I am more likely to remember `yarn feature-search`. 😁